examples/eth1_del_route.yml

```diff
@@ -0,0 +1,16 @@
+---
+interfaces:
+  - name: eth1
+    type: ethernet
+    state: up
+    ipv4:
+      address:
+      - ip: 192.0.2.251
+        prefix-length: 24
+      dhcp: false
+      enabled: true
+
+routes:
+  config:
+  - next-hop-interface: eth1
```

Comment:
IMHO this should at least specify some more criteria for the specific route or the file should be called `eth1_delete_all_routes.yml` to indicate that it operates on all routes.